GPU/GPUTracking/Merger/GPUTPCGMPropagator.cxx

```diff
@@ -651,6 +652,9 @@ GPUd() int GPUTPCGMPropagator::Update(float posY, float posZ, int iRow, const GP
       rejectChi2 = 1;
     } else {
       int retVal = InterpolateReject(posY, posZ, clusterState, rejectChi2, inter, err2Y, err2Z);
+      if (o2::utils::DebugStreamer::checkStream(o2::utils::StreamFlags::streamRejectCluster)) {
+        o2::utils::DebugStreamer::instance()->getStreamer("debug_InterpolateReject", "UPDATE") << o2::utils::DebugStreamer::instance()->getUniqueTreeName("tree_InterpolateReject").data() << "mAlpha=" << mAlpha << "iRow=" << iRow << "posY=" << posY << "posZ=" << posZ << "clusterState=" << clusterState << "rejectChi2=" << rejectChi2 << "interpolationhit=" << *inter << "refit=" << refit << "retVal=" << retVal << "err2Y=" << err2Y << "err2Z=" << err2Z << "track=" << mT << "\n";
```

Comment:
Hello @Mathias.
here I would like to keep also return value. it is not in the current streamer

Reply:
Indeed in session we found it is already there

Reply:
You can use `(size_t)mT` as unique track ID

Reply:
I am just looking at `(size_t)mT` and it seems to be the same for every track. There are only differences if I compare the `(size_t)mT` for different CPU threads. Are you sure it can be used as unique track ID?